lib/features/boost/config.dart

```diff
@@ -55,18 +77,47 @@ class BoostConfigState extends ConsumerState<BoostConfig> {
           children: [
             // Space to the left of the configs.
 
-            const SizedBox(width: 5),
+            configLeftSpace,
 
             // The BUILD button.
 
             ActivityButton(
-              onPressed: () {
-                showUnderConstruction(context);
+              onPressed: () async {
```

Comment:
Need to also pass the pageControllerProvider to the ActivityButton so we get the NextPage behaviour after pressing the button.